CONTRIBUTING.md

```diff
@@ -7,75 +7,74 @@ The following are a set of guidelines for contributing to the project.
+Once a contribution is being reviewed, the Core Engineering Team will apply a relevant label to the associated issue. You can see our [label list on GitHub](https://github.com/adobe/aem-core-wcm-components/labels) to better understand what each label means.
 
-### Joining Developer Discussions 💬
-
-If you have thoughts on the code, the general architecture of the project, or would like some initial feedback on a feature you are thinking of coding, you can visit the [Core Components Developer Mailing List](https://groups.google.com/forum/#!forum/aem-core-components-dev) on Google Groups, or [subscribe via email](mailto:aem-core-components-dev+subscribe@googlegroups.com).
-
-### Reporting Bugs 🐛
-
-##### Before Reporting a Bug 
+### Reporting and Fixing Bugs 🐛
 
+#### Before Reporting a Bug 
 * Have a quick search through the currently open [bug reports](https://github.com/adobe/aem-core-wcm-components/labels/bug) to see if the issue has already been reported.
 * Ensure that the issue is repeatable and that the actual behavior versus the expected results can be easily described.
 * Check that the issue you are experiencing is related to the Core Components project. It may be that the problem derives from AEM itself, typically editor code, rather than the Core Components. If you're not sure, then feel free to report the issue anyway and the committers will clarify for you. Issues in the product can be reported via [Adobe Enterprise Support](https://helpx.adobe.com/contact/enterprise-support.ec.html).
 
-##### Filing a Bug
-
+#### Filing a Bug
 1. Visit our [issue tracker on GitHub](https://github.com/adobe/aem-core-wcm-components/issues).
 1. File a `New Issue` as a `Bug Report`.
 1. Ensure your issue follows the [issue report guidelines](#issue-report-guidelines).
 1. Thanks for the report! The committers will get back to you in a timely manner, typically within one week.
 
-### Requesting Features 🚀
-
-##### Before Requesting a Feature
+#### Fixing a Bug
+If you have a fix ready for a bug, submit a [pull request](#contributing-code) and reference it in the associated issue.
 
+### Submitting Features 🚀
 * Have a quick search through the currently open [enhancement](https://github.com/adobe/aem-core-wcm-components/labels/enhancement) and [request to comment](https://github.com/adobe/aem-core-wcm-components/labels/rtc) issues to see if the idea has already been suggested. If it has, you may still have a slightly different requirement that isn't covered, in which case, feel free to comment on the open issue. 
```

Comment:
@glasert - are we missing here a point that feature submissions (i.e. creating a feature request issue) in the repo should be accompanied by a pull request?

Reply:
@richardhand I agree and I wonder whether we want to mention this together with the information about the contrib repo?
```suggestion
**Please Note**: New components and experimental features should always go to [aem-core-wcm-component-contrib]("") repository by default.
All feature submissions should be accompanied by a [pull request]("#contributing-code-")
```

Reply:
@glasert - I think I would make these two separate points in this section, rather than part of the "have a quick search through" bullet point.

For the note on the contrib repo, I think we could word it slightly differently:
```
"New experimental components and features should always go to"
``` 

not all new components would go directly to contrib if they're acceptable in the core.